starter-code/src/clue.js

```diff
@@ -1,94 +1,122 @@
 // Characters
-/*
-mrGreen
-first_name:   Jacob
-last_name:    Green
-color:        green
-description:  He has a lot of connections
-age:          45
-image:        https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg
-occupation:   Entrepreneur
 
-drOrchid
-frst_name:   Doctor
-lst_name:    Orchid
-color:        white
-description:  PhD in plant toxicology. Adopted daughter of Mr. Boddy
-age:          26
-image:        http://www.radiotimes.com/uploads/images/Original/111967.jpg
-ocupation:   Scientist
+var mrGreen = {
+    first_name:   "Jacob",
+    last_name:    "Green",
+    color:        "green",
+    description:  "He has a lot of connections",
+    age:          45,
+    image:        'https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg' , 
+    occupation:   "Entrepreneur"}
+    
+    var drOrchid = {
+    frst_name:   "Doctor",
+    lst_name:    "Orchid",
```

Comment:
you made a small typo here. You missed the second letter from the 'first' and the 'last'